app/assets/javascripts/create.js

```diff
@@ -0,0 +1,32 @@
+$(() => {
+    createIdea()
+})
```

Comment:
So, as you probably know, this will execute when the document is loaded by the browser and ready to go. One thing you could do here is go ahead and query for all of the DOM elements you plan on working with. For example, you query for `$('#title-field')` and `$('#body-field')` every time you go to create an idea. This is extra work for jQuery. 

Alternatively, you could do something like:

```js
$(() => {
    var $titleField = $('#title-field');
    var $bodyField = $('#body-field');

    createIdea()
})
```

The dollar signs aren't important. I just use them to signify that the variables represent jQuery collections. There are also many more than just these two selectors that you can and should cache. Querying the DOM is kind of like ActiveRecord queries. Use as many as you need, but as few as you can get away with.